components/org.wso2.carbon.identity.api.dispatcher/src/main/webapp/WEB-INF/beans.xml

```diff
@@ -60,6 +61,7 @@
             <bean class="org.wso2.carbon.identity.api.server.permission.management.v1.PermissionManagementApi"/>
             <bean class="org.wso2.carbon.identity.api.server.keystore.management.v1.KeystoresApi"/>
             <bean class="org.wso2.carbon.identity.api.server.idp.v1.IdentityProvidersApi"/>
+            <bean class="org.wso2.carbon.identity.api.server.oidc.scope.management.v1.OidcApi"/>
```

Comment:
```suggestion
            <bean class="org.wso2.carbon.identity.api.server.oidc.scope.management.v1.OIDCApi"/>
```
Package naming?

Reply:
This is a generated class hence we cant change I think..
 https://github.com/wso2/identity-api-server/blob/8baa8d1f9eb0a548bd080c794d000735241021df/components/org.wso2.carbon.identity.api.server.oidc.scope.management/org.wso2.carbon.identity.api.server.oidc.scope.management.v1/src/gen/java/org/wso2/carbon/identity/api/server/oidc/scope/management/v1/OidcApi.java#L39